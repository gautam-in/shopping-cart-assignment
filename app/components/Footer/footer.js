import FooterStyle from './footer.styles';
export default function Footer() {
  return (
    <FooterStyle>
      <p>Copyright &copy; 2011-2018 Sabka Bazaar Grocery Supplies Pvt. Ltd. </p>
    </FooterStyle>
  );
}
