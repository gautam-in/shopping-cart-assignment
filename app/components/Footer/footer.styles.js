import styled from 'styled-components';

const FooterStyle = styled.footer`
  background-color:${props => props.theme.lightgrey};
  position: fixed;
  bottom: 0;
  z-index: 1;
  font-size: 0.8rem;
  height: ${props => props.theme.footerHeight}px;
  display: flex;
  align-items: center;
  justify-content: center;
  width: 100%;

  @media only screen and (max-width: 767px) {
    margin: 0;
    font-size: 0.6rem;
    height:${props => props.theme.footerHeight -10}px;
    width: 100%;
  }
`;

export default FooterStyle;
