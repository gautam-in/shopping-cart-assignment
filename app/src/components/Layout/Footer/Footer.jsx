import React from "react";
import style from './footer.module.css'
const Footer = (props) => {
    const date = new Date()
    return (
        <div
            className={style.footerContainer}
        >
            <div
            className={style.footerContent}
                dangerouslySetInnerHTML={{ "__html": `Copyright &#169; 2011 - ${date.getFullYear()} Sabka Bazaar Grocery Supplies Pvt. Ltd.` }} />
        </div>
    )
}

export default Footer;