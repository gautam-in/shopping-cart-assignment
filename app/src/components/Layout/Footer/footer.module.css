.footerContainer{
    margin: 2% 0;
    background-color: rgb(238, 237, 237);
    padding: 1%;
}
.footerContent{
    margin: 0 15%;
    color: black;
    font-weight: 600;
    text-align: left;
}