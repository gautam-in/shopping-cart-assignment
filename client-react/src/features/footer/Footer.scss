@import '../../Variables.scss';

footer {
    background-color: $grey-1;
    position: fixed;
    bottom: 0;
    width: 100%;
    .app-container {
        padding: 20px 0!important;
    }
}