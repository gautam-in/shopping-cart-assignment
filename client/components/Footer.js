import { FooterStyle } from "../styles/FooterStyle";

const Footer = () => {
  return (
    <FooterStyle>
      <div>
        Copyright &copy; 2011-2018 Sabka Bazaar Grocery Supplies Pvt Ltd
      </div>
    </FooterStyle>
  );
};

export default Footer;
