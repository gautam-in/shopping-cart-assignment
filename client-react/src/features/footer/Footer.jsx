import React from 'react';
import './Footer.scss';

export const Footer = () => {
  return (
    <footer>
        <div className='container app-container'>
          <div className='row'>
            <div className='col-lg-9 offset-lg-3 col-md-8 offset-md-4'>
              <p className='d-flex justify-content-center justify-content-md-start mb-0'>Copyright &copy; 2011-2018 Sabka Bazaar Grocery Supplies Put Ltd</p>
            </div>
          </div>
        </div>
    </footer>
  )
}
