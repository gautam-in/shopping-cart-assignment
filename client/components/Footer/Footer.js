import React from "react";
import "./Footer.scss";

const Footer = () => {
  return (
    <div className="footer">
      copyright &copy; 2021-2022 Sabka bazaar grocery supplies Pvt. Ltd.
    </div>
  );
};

export default Footer;
