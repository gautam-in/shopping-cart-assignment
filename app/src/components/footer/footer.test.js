import Footer from './footer';
import renderer from 'react-test-renderer';

describe('test footer snapshot', () => {
    it('test footer', () => {
        const component = renderer.create(<Footer />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
})