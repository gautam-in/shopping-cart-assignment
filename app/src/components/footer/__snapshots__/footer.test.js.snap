// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test footer snapshot test footer 1`] = `
<div
  className="footer"
>
  <div
    className="container"
  >
    <p>
      Copyright Â© 2011-2022 Sabka Bazaar Grocery Supplies Pvt Ltd.
    </p>
  </div>
</div>
`;
